public/electron.js

```diff
@@ -94,6 +125,32 @@ if (process.platform === 'darwin') {
 // Some APIs can only be used after this event occurs.
 app.on('ready', createWindow)
 
+function startSystray() {
+  tray = new Tray(path.join(__dirname, trayIcon));
+  const contextMenu = Menu.buildFromTemplate([
+    {
+      label: 'Show/Hide', 
```

Comment:
Why don't we adjust the label according to the current state? It would be either `Show` or `Hide`.

Reply:
I prefer `Hide Wallet` and `Show Wallet`.

Reply:
Okay, you're right.